frontend/javascripts/dashboard/explorative_annotations_view.tsx

```diff
@@ -382,14 +383,10 @@ class ExplorativeAnnotationsView extends React.PureComponent<Props, State> {
   };
 
   renameTracing(tracing: APIAnnotationInfo, name: string) {
-    editAnnotation(tracing.id, tracing.typ, { name })
-      .then(() => {
-        Toast.success(messages["annotation.was_edited"]);
-        this.updateTracingInLocalState(tracing, (t) => update(t, { name: { $set: name } }));
-      })
-      .catch((error) => {
-        handleGenericError(error as Error, "Could not update the annotation name.");
-      });
+    Store.dispatch(
```

Comment:
this can't work because there's no save saga in the dashboard. is there no non-update-action route to do the editing anymore?
@fm3 @MichaelBuessemeyer

Reply:
There is not. Do you think this is important? Then we could add a route to the tracingstore that creates and submits the update action in the backend

Reply:
I definitely want to avoid that the save saga is executed in the dashboard. Therefore, we need some code that handles the update action in another way. It could be in front-end or back-end. In my biased opinion, I would prefer the back-end variant, because otherwise the front-end would need to download the `version` first, if I see it correctly.
Would it be okay with you to go with the back-end variant?

Reply:
Yes, should be fine. Would it be ok to do this on the tracingstore side? Does the frontend have the tracingstore URI and a token at that time?

Reply:
> Does the frontend have the tracingstore URI and a token at that time?

Unfortunately, no. The front-end has only the `APIAnnotationInfo` object. Ideally, there would still be the old `/api/annotations/${annotationType}/${annotationId}/edit` route.

Reply:
Ok. That route still exists, but it is not currently connected to the update actions. Looks like it currently edits postgres, but that may soon be overridden by what is reported from the tracingstore. I think I’ll be able to connect the two sides. I added it to my todo list. You can switch this back in the frontend use that route already.